setup.py

```diff
@@ -38,9 +38,7 @@
         'numpy>=1.7',
         'scipy>=0.13',
         'matplotlib',
-        'ipywidgets',
         'properties[math]',
```

Comment:
while we are at it is it worth updating this to 
'properties' and 'vectormath'? I think it clarifies the installation a bit